src/extension/ncwms/src/main/java/au/org/emii/geoserver/wms/Ncwms.java

```diff
@@ -104,7 +109,15 @@ private void proxyWmsRequest(HttpServletRequest request, HttpServletResponse res
         try {
             @SuppressWarnings("unchecked")
             Map<String, String[]> wmsParameters = new HashMap<String, String[]>(request.getParameterMap());
-            wmsParameters.remove("TIME");
+
+            // Some collections such as SRS are indexed with a timestamp which doesn't match
```

Comment:
Should the harvester be changed to record instantaneous dates, as well as intervals when that's appropriate? Assuming I understand the issue.